ClientApp/viking-vault/src/display_user_list/ResponseModal.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './ResponseModal.css';
+
+interface IResponseModalProps{
+    open: boolean;
+    closeModal: any;
```

Comment:
You can specify function types, like this `func: (x: number, y: number) => number;`, you can use `void` return type if the function just does something